### PR TITLE
fix(documentation): fix grammar mistakes, typos and references.

### PR DESCRIPTION
**Pull request by asafkorem**
Fix grammar mistakes, typos and references inspected by WebStorm, using WebStorm's fix suggestions.

### CODE REVIEW

File: docs/APIRef.LaunchArgs.md [hunk 1]
```diff
@@ -48,4 +48,4 @@ On iOS, the specified launch arguments are passed as the process launch argument
 
 On Android, the launch arguments are set as bundle-extra’s into the activity’s intent. It will therefore be accessible on the native side via the current activity as: `currentActivity.getIntent().getBundleExtra("launchArgs")`.
 
-Further handling of these launch arguments is up to the user’s responsibility and is out of scope for Detox.
+Further, handling of these launch arguments is up to the user’s responsibility and is out of scope for Detox.
```
Comment (noomorph) [inline docs/APIRef.LaunchArgs.md:51]: should be: The further - and without that comma suggestion

File: docs/APIRef.MockingUserNotifications.md [hunk 2]
```diff
@@ -171,7 +171,7 @@ activity.getIntent().getBundleExtra("userDataObj").getString("cascadedKey") == "
 
 ##### Handling at Runtime
 
-Note that on Android, data delivered through an intent at runtime, is typically received in your activity’s [`onNewIntent`](https://developer.android.com/reference/android/app/Activity#onNewIntent\(android.content.Intent\)) callback. Be sure to consider what should be done in order to handle this type of a use case in your app: Namely, that `setIntent()` should be called in order for the data to be later available in your app through `getIntent()`, as explained earlier.
+Note that on Android, data delivered through an intent at runtime, is typically received in your activity’s [`onNewIntent`](https://developer.android.com/reference/android/app/Activity#onNewIntent\(android.content.Intent\)) callback. Be sure to consider what should be done in order to handle this type of use case in your app: Namely, that `setIntent()` should be called in order for the data to be later available in your app through `getIntent()`, as explained earlier.
```
Comment (noomorph) [inline docs/APIRef.MockingUserNotifications.md:174]: Miscorrection. 'A' article can be used with 'use case'. Either, 'the'.

File: docs/Guide.RunningOnCI.md [hunk 3]
```diff
@@ -174,7 +174,7 @@ workflows:
...
+GitLab is also a popular git management service which also include a built-in CI system. They provide free runner-up to 2000 minutes for private projects, however, the runners provided by them cannot be used to run Detox due to the lack of KVM support (in order to run Android Emulators). You can, instead, [create your own runner](https://docs.gitlab.com/ee/ci/runners/README.html) with KVM support. Some example of cloud providers offering this are: [Digital Ocean](https://www.digitalocean.com/products/droplets/), AWS (with [C5 instance types](https://aws.amazon.com/ec2/instance-types/c5/)), [Google Cloud](https://cloud.google.com/compute/docs/instances/enable-nested-virtualization-vm-instances) and [Azure](https://docs.microsoft.com/en-us/azure/virtual-machines/windows/nested-virtualization)
```
Comment (noomorph) [inline docs/Guide.RunningOnCI.md:177]: miscorrection

**Review by noomorph — CHANGES_REQUESTED** [review]

**Comment by asafkorem** [comment]
@noomorph I missed your comments before merging this. I'll fix it in a follow-up PR.

**Pull request merged**